### PR TITLE
Adds php 8.3

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

File: dev/Dockerfile [hunk 1]
```diff
@@ -7,8 +7,9 @@ ARG ARCH=amd64
 USER root
 
 RUN apk --update --no-cache add \
-      php${PHP_VERSION}-xdebug \
-      php${PHP_VERSION}-spx
+      php${PHP_VERSION}-xdebug
+
+RUN if [ "$PHP_VERSION" != "8.3" ]; then apk --update --no-cache add; fi
```
Comment (nickschuch) [inline dev/Dockerfile:12]: Missing the packages?

**Comment by nickschuch** [comment]
We will also need to ommit the spx config file and newrelic from the build.

```
$ docker run -it docker.io/skpr/php-cli:8.3-dev-v2-latest-amd64 bash
skpr@8830d9b106bf:data $ php -m
PHP Warning:  PHP Startup: Unable to load dynamic library 'newrelic.so' (tried: /usr/lib/php/modules/newrelic.so (Error loading shared library /usr/lib/php/modules/newrelic.so: No such file or directory), /usr/lib/php/modules/newrelic.so.so (Error loading shared library /usr/lib/php/modules/newrelic.so.so: No such file or directory)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library 'newrelic.so' (tried: /usr/lib/php/modules/newrelic.so (Error loading shared library /usr/lib/php/modules/newrelic.so: No such file or directory), /usr/lib/php/modules/newrelic.so.so (Error loading shared library /usr/lib/php/modules/newrelic.so.so: No such file or directory)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'spx.so' (tried: /usr/lib/php/modules/spx.so (Error loading shared library /usr/lib/php/modules/spx.so: No such file or directory), /usr/lib/php/modules/spx.so.so (Error loading shared library /usr/lib/php/modules/spx.so.so: No such file or directory)) in Unknown on line 0

Warning: PHP Startup: Unable to load dynamic library 'spx.so' (tried: /usr/lib/php/modules/spx.so (Error loading shared library /usr/lib/php/modules/spx.so: No such file or directory), /usr/lib/php/modules/spx.so.so (Error loading shared library /usr/lib/php/modules/spx.so.so: No such file or directory)) in Unknown on line 0
Segmentation fault (core dumped)
skpr@8830d9b106bf:data $ 
```

**Comment by kimpepper** [comment]
Getting segfaults. Need to debug further.

**Comment by kimpepper** [comment]
Segfaults are possibly due to https://github.com/xdebug/xdebug/pull/934 
Disabling xdebug extension in php 8.3 for now.

**Pull request merged**